cli/src/pcluster/validators/iam_validators.py

```diff
@@ -57,6 +59,33 @@ def _validate(self, policy: str):
             self._add_failure(str(e), FailureLevel.ERROR)
 
 
+class IamResourcePrefixValidator(Validator):
+    """
+    Iam Resource Prefix Validator.
+
+    Verify if the Resource Prefix belongs in the permission boundary if given.
```

Comment:
I think this comment is wrong. The goal of this validator is to verify that the ResourcePrefix is compliant with the expected format

Reply:
I will change the Comment.